packages/migration/src/types/element.ts

```diff
@@ -40,6 +40,17 @@ export interface FontMetrics {
   lGap: number;
 }
 
+export interface Font {
+  family: string;
+  fallback?: string[];
+  service?: string;
+  metrics?: FontMetrics;
+}
+
+export interface Fonts {
```

Comment:
Do we have a currently existing `Font` type somewhere outside of the migration package that'd need an update as well?

Reply:
Not that I'm aware of but yes we could move this out into types?

Reply:
Removed the migration so this isn't in use now.